recipes/genera/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Exit on any error
+set -ex
+
+# Ensure all main scripts are executable
+chmod +x genEra Erassignment hmmEra tree2ncbitax FASTSTEP3R test_installation.sh
+
+# Clone the secondary repository and move the main script
+git clone https://github.com/caraweisman/abSENSE.git
+mv abSENSE/Run_abSENSE.py .
+chmod +x Run_abSENSE.py
```

Comment:
You are right that the git clone command should be removed, but instead of adding the missing dependency as a source, a separate recipe should be created for the dependency in a new PR and then the dependencies updated in this recipe.

Reply:
Hi @aliciaaevans, thank you for the review. This particular script doesn’t have a release version as I believe it’s part of a publication’s codebase. I’ve submitted a request on their GitHub for a release, but if they don’t create one, would it be possible to proceed with `git clone` despite the recommendation to avoid it?

Reply:
I have added the required recipe for abSENSE (it has been merged https://anaconda.org/bioconda/absense), removed the corresponding section from GenEra, and made the necessary adjustments